.circleci/config.yml

```diff
@@ -68,7 +68,7 @@ jobs:
             fi
             STATUS=$?;
             if [[ ! -z $FORCE_BUILD ]] ; then
-              echo 'FORCE_BUILD active!'
+              echo 'FORCE_BUILD active! The build not fails if the tests are failing.'
```

Comment:
this is grammatically wrong. Also we should include the original status code. see my original review.

Reply:
I hope you like more the second version.